server/src/server.rs

```diff
@@ -234,6 +236,40 @@ impl Server {
         Ok(())
     }
 
+    async fn control(
+        &self,
+        session: SessionId,
+        packet: proto::Control,
+        from: SocketAddr,
+    ) -> ServerResult<()> {
+        log::info!("Control packet from: {}", from);
+
+        if let proto::Control {
```

Comment:
Maybe `match` and `log::debug` on `_ (other control packets)` by default?

Reply:
Check new version